debug/org.eclipse.ui.console/src/org/eclipse/ui/internal/console/ConsoleUIPreferenceInitializer.java

```diff
@@ -29,6 +29,8 @@ public void initializeDefaultPreferences() {
 		IPreferenceStore prefs = ConsolePlugin.getDefault().getPreferenceStore();
 		prefs.setDefault(IConsoleConstants.P_CONSOLE_AUTO_SCROLL_LOCK, true);
 		prefs.setDefault(IConsoleConstants.P_CONSOLE_WORD_WRAP, false);
+		prefs.setDefault(IConsoleConstants.AUTO_PIN_ENABLED_PREF_NAME, false);
```

Comment:
We should enable by default, it makes no sense to keep it as is. 
The dialog shown is enough to explain the user the new behavior.